rails/app/views/restoration_activity_log_entries/index.html.erb

```diff
@@ -5,7 +5,7 @@
     <div class="col-4">
       <div class="card text-center">
         <div class="card-body restoration-index">
-          <h5 class="card-title"><%= entry.nursery_table.name %></h5>
+          <h5 class="card-title"><%= entry.nursery_table&.name %></h5>
```

Comment:
Can we move this (`&.`) logic into the model, say as a `table_name` (maybe use a delegator with allow_nil) so that we don't have to manage to replicate this logic in every view?

Reply:
yep. That sounds more readable too. Working on this now.